.teamcity/_self/projects/WPComTests.kt

```diff
@@ -255,6 +271,7 @@ fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): E2E
 			)
 			param("env.AUTHENTICATE_ACCOUNTS", "gutenbergSimpleSiteEdgeUser,gutenbergSimpleSiteUser,coBlocksSimpleSiteEdgeUser,simpleSitePersonalPlanUser")
 			param("env.VIEWPORT_NAME", "$targetDevice")
+			if (atomic) Parametrized().param("env.TEST_ON_ATOMIC", "true")
```

Comment:
I'm not familiar with `Parametrized`, I assume that in the case where the `if (atomic)` call evaluates to `false` the param value will not be set, is that correct?

Reply:
That's what I hope will happen! But I'm not familiar with it either.

Reply:
Actually, I don't need to create the instance from `Parametrized` manually here. Just calling `param` seems to be enough. I'll push a fix.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/62037/commits/4913cb2c6626d53a914ef2a3bd6cb4b431b98adb.